fs/cutoffmode.go

```diff
@@ -0,0 +1,49 @@
+package fs
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/pkg/errors"
+)
+
+// CutoffMode describes the possible delete modes in the config
+type CutoffMode byte
```

Comment:
This file should have a test file `cutoffmode_test.go` - eg see https://github.com/rclone/rclone/blob/master/fs/parseduration_test.go

This test file should assert the required interface, eg

https://github.com/rclone/rclone/blob/da5cbc194a6fa337e0ae162da5c89605f8347232/fs/parseduration_test.go#L14